pkg/collector/edp/payload.go

```diff
@@ -0,0 +1,46 @@
+package edp
+
+import "github.com/kyma-project/kyma-metrics-collector/pkg/resource"
+
+type Payload struct {
+	RuntimeID    string                  `json:"runtime_id"           validate:"required"`
+	SubAccountID string                  `json:"sub_account_id"       validate:"required"`
+	ShootName    string                  `json:"shoot_name"           validate:"required"`
+	Timestamp    string                  `json:"timestamp"            validate:"required"`
+	Compute      resource.EDPMeasurement `json:"compute"              validate:"required"`
+	Networking   *Networking             `json:"networking,omitempty"`
+}
+
+type Networking struct {
+	ProvisionedVnets int `json:"provisioned_vnets" validate:"numeric"`
+	ProvisionedIPs   int `json:"provisioned_ips"   validate:"numeric"`
+}
+
+func newPayload(runtimeID, subAccountID, shootName, timeStamp string, EDPMeasuremnets []resource.EDPMeasurement) Payload {
```

Comment:
unexported constructor for exported type?

Reply:
I unexported the `payload` struct as it is only used in the same package, so no need to export it.